autosklearn/estimators.py

```diff
@@ -664,7 +665,9 @@ def fit(self, X, y,
         # Before running anything else, first check that the
         # type of data is compatible with auto-sklearn. Legal target
         # types are: binary, multiclass, multilabel-indicator.
-        target_type = type_of_target(y)
+        target_type = type_of_target(
+            InputValidator().validate_target(y, is_classification=True)
+        )
```

Comment:
Sorry, can you please elaborate on this comment?

Reply:
I meant that replacing:
```python
if target_type in ['multiclass-multioutput', 'continuous', 'continuous-multioutput', 'unknown',]:
    raise ValueError("classification with data of type %s is not supported" % target_type)
```
by
```python
if target_type not in ['binary', 'multiclass']:
    raise ValueError("classification with data of type %s is not supported" % target_type)
```
So we check what is allowed instead of checking what is forbidden.